### DISCUSSION TITLE
Why is an 'x' on this file?

### DISCUSSION THREAD

**Discussion by andrei-edward-popa** [zephyrproject-rtos/zephyr#42753] [General]
https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/serial/uart_rpi_pico.c

Hi! If you look at this file it has an 'x' on top and an error in twister build that says: Error: Value cannot be null. (Parameter 'ContainerId').

Do you know what the issue is?

**Comment by stephanosio** [comment]
because the CI run was cancelled (when there is a new commit on the main, any on-going CI runs on the main will be cancelled).
